src/test/index.ts

```diff
@@ -0,0 +1,20 @@
+// This file is used by VS Code's default test runner to configure Mocha before the test run.
+
+/* tslint:disable:no-var-keyword no-var-requires */
+var testRunner = require("vscode/lib/testrunner");
+/* tslint:enable:no-var-keyword no-var-requires */
+
+let mochaOptions: any = {
+    ui: "tdd",
```

Comment:
There may be small semantic differences in the approach of BDD vs TDD in general, as discussed in [this SO question](http://programmers.stackexchange.com/questions/135218/what-is-the-difference-between-writing-test-cases-for-bdd-and-tdd), but as far as Mocha test authoring is concerned, they function pretty much identically. 

The only difference is in the name of the different methods:

> 
> The BDD interface provides describe(), it(), before(), after(), beforeEach(), and afterEach().
> 
> The TDD interface provides suite(), test(), suiteSetup(), suiteTeardown(), setup(), and teardown().